python/pyspark/ml/wrapper.py

```diff
@@ -263,7 +282,14 @@ def _fit_java(self, dataset):
 
     def _fit(self, dataset):
         java_model = self._fit_java(dataset)
-        return self._create_model(java_model)
+        model = self._create_model(java_model)
+
+        # SPARK-10931: This is a temporary fix to allow models to own params
+        # from estimators. Eventually, these params should be in models through
+        # using common base classes between estimators and models.
+        model._create_params_from_java()
```

Comment:
So right now this would apply to all of the models, would it make sense to make it so that we can selectively move the params forward one at a time?